api/tests/Feature/ApplicantFilterTest.php

```diff
@@ -98,73 +100,6 @@ protected function filterToCreateInput(ApplicantFilter $filter)
         return $input;
     }
 
-    /**
```

Comment:
Is it worth repointing these tests at `poolCandidateSearchRequest.applicantFilter` instead of removing them?  Or am I misunderstanding how this works?

Reply:
I noticed that the test `testFilterCanBeStoredAndRetrievedWithoutChangingResults()` was querying an applicant filter so I thought why not move the test there and save some space. However, I think your right looking back at it now since it's querying for all the fields which makes for a stronger test. I'll make the changes now 👍